slides/workshop_rap_for_managers_basics.qmd

```diff
@@ -257,15 +275,14 @@ Recyclable code is code that can be used in multiple projects. This means that y
 
 # What we expect from managers
 
-::: {.column width="40%"}
-- You should be familiar with our RAP standards, and we expect managers to ensure that their teams are following these standards
-- When your team reaches "baseline" (all elements of Good + Great), they will undergo an audit to ensure that the standards have been met
-- We then expect teams to begin working towards Best practice levels
+::: {.column width="45%"}
+- You should have sufficient levels of technical skills (R, SQL, Git) relevant to the work of your team to allow you to act as a technical leader and build capability in your team and the wider analyst community
```

Comment:
Consider rewording to something like: 

That you have sufficient confidence and ability relevant to the work of your team to allow you to act as an effective technical leader, to build capability and resilience within your team, challenge processes to ensure high quality / value for effort and act as an effective support where needed through advice and peer review.